### DISCUSSION TITLE
[📖 Docs]: <title> Why is there a different implementation for the html a element and the other html elements concerning selectors ?

### DISCUSSION THREAD

**Discussion by akfoury** [webdriverio/webdriverio#8598] [General]
### Pre-check

- [X] I know I can edit the docs but prefer to file this issue

### Describe the improvement

Missing documentation

### Description of the improvement / report

For selecting an element with a certain text you have to do  $('span=welcome')  whereas  for  selecting  a  link  element  (a  tag) 
 you  have  to  do  like  this  $('=welcome'). This small difference can lead to some confusion among developers.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by christian-bromann** [comment]
Selecting links by text is a native WebDriver selector strategy while selecting html elements by text requires a conversion into xpath by WebdriverIO. This is documented well on our [selectors page](https://webdriver.io/docs/selectors). From the devx perspective, yes this might be debatable. Any suggestions for improvements are welcome!
